pressbooks/pressbooks-book
# Restore 'SpecialElite' font (used by Leonard)

Fix for https://github.com/pressbooks/pressbooks-leonard/issues/11

To test:
1. Open a book and switch to the Leonard theme
2. Generate EPUB and PDF exports. Confirm that they appear with expected styling and no unusual errors.
